src/models/user_data_processing.ts

```diff
@@ -24,33 +24,32 @@ export const USER_DATA_PROCESSING_COLLECTION_NAME = "user-data-processing";
 export const USER_DATA_PROCESSING_MODEL_PK_FIELD = "fiscalCode" as const;
 export const USER_DATA_PROCESSING_MODEL_ID_FIELD = "userDataProcessingId" as const;
 
-/**
- * Ensure UserDataProcessing IDs are in correct shape
- * The unique identifier of a user data processing request identified by concatenation offiscalCode - choice
- */
-export type UserDataProcessingId = t.TypeOf<typeof UserDataProcessingId>;
-export const UserDataProcessingId = t.brand(
-  t.string,
-  (
-    s: string
-  ): s is t.Branded<string, { readonly IUserDataProcessingIdTag: symbol }> => {
+interface IUserDataProcessingIdTag {
```

Comment:
Wasn't better to maintain branded type that is a more recent functionality than refinement? Refinement is deprecated in new io-ts versions.